app/build.gradle

```diff
@@ -7,7 +7,7 @@ android {
 
     defaultConfig {
         applicationId "org.wordpress.aztec"
-        minSdkVersion 16
```

Comment:
What was the rationale for updating the minSdk version here?

Reply:
I have this error with sdkmin 16: Manifest merger failed : uses-sdk:minSdkVersion 16 cannot be smaller than version 18 declared in library [org.wordpress:utils:1.30.1] /Users/starship/.gradle/caches/transforms-1/files-1.1/utils-1.30.1.aar/dd4efbd0770e093392f120893bd164d1/AndroidManifest.xml as the library might be using APIs not available in 16